### DISCUSSION TITLE
Image quality using Origin.

### DISCUSSION THREAD

**Discussion by holden** [swellstores/community#383] [Ideas]
Hi,

I'm just noticing a significant image quality issue one some of our bigger images when using the Origin template on pages created using the Swell Editor. We set the quality from 80 to 98 in `visualmedia.js` which appears to control these, but it doesn't seem to be the determining factor:

```
    quality: {
      type: Number,
      default: 98,
    },
```

We've tried uploading JPEGs, PNGs, and WEBPs, it doesn't seem to make a difference, to take an extreme example it will downscale a 3MB 4,000x1,200 pixel PNG down to 98k 1440 or convert a 486k WEBP to a 70kb JPEG.

Example page: https://beta.teamups.co/halloween/

How can we control the quality and format?

![1](https://github.com/swellstores/community/assets/48241/da5d5589-1192-4ab7-b482-43f89e825a82)
![1387d2bfa3c2a7866febc58906d5bead](https://github.com/swellstores/community/assets/48241/1da6714b-ed50-4671-8a3c-13149fe1ed95)

Thanks,

**Comment by ericingram** [comment]
@holden it looks like we are setting quality explicitly in the CDN processor and there's no option to modify it currently. This might have changed in the last few months as we had another functional layer in the CDN that was disabled.

We can consider supporting the quality option as an improvement (not sure if it was actually working before or not) @logeshswell
